Python/chapter01/1.3 - URLify/lomeli_1.3.py

```diff
@@ -0,0 +1,14 @@
+# Write a method to replace all spaces in a string with '%20'
+# "Mr John Smith     ", 13 => "Mr%20John%20Smith"
+
+def Urlify(string, size):
+    temp = []
+    url = '%20'
+    for i in range(size):
```

Comment:
it might be more [Pythonic](https://docs.python-guide.org/writing/style/) to use

```python
for c in string[:size]:
```